Blockzilla/UIComponents/UIConstants.swift

```diff
@@ -335,6 +335,11 @@ struct UIConstants {
         static let defaultBrowserOnboardingViewTopButtonTitleV2 = NSLocalizedString("Onboarding.DefaultBrowser.TopButtonTitle.V2", value: "Set as Default Browser", comment: "Text for a label that indicates the title of the top button from the default browser onboarding screen version 2.")
         static let defaultBrowserOnboardingViewBottomButtonTitleV2 = NSLocalizedString("Onboarding.DefaultBrowser.BottomButtonTitle.V2", value: "Skip", comment: "Text for a label that indicates the title of the bottom button from the default browser onboarding screen version 2.")
         static let tooltipBodyTextForTrashIconV2 = NSLocalizedString("TooltipBodyText.TrashIcon.V2", value: "Tap here to trash it all — history, cookies, everything — and start fresh on a new tab.", comment: "This is the body text that is displayed for the Trash icon tooltip")
+        static let titleShowMeHowOnboardingV2 = NSLocalizedString("ShowMeHowOnboarding.Title.V2", value: "Turn on Sync", comment: "This is the title text that is displayed in the Show Me How Onboarding Screen")
```

Comment:
Double checking: is Sync uppercase because titles on iOS require title case, or because "Sync" is a feature name? The latter is not true anymore, so making sure that's not the case.

Reply:
Correct, we don't have a sync feature. That might be a mistake from UX. I will double check with them.

Reply:
New string will be "Add a Focus Widget"